### PR TITLE
Use csvutil.BetterNewReader

### PR DESCRIPTION
**Pull request by cosmonautspiff**
Uses BetterNewReader function from "github.com/antha-lang/toolbox/csvutil" to deal with carriage returns.

### CODE REVIEW

File: target/mixer/parse_plate.go [hunk 1]
```diff
@@ -2,17 +2,16 @@ package mixer
 
 import (
 	"context"
-	"encoding/csv"
 	"fmt"
+	"github.com/antha-lang/antha/antha/anthalib/wtype"
```
Comment (ddn0) [inline target/mixer/parse_plate.go]: looks like you are not using gofmt or goimports.

Comment (cosmonautspiff) [reply target/mixer/parse_plate.go]: fmt is being used in parse_plate.go. What don't I understand?

**Comment by ddn0** [comment]
Agreed, but that test case probably should be in the github.com/antha-lang/toolbox rather than here.

**Comment by chr1sgrant** [comment]
why not in both?

**Comment by cosmonautspiff** [comment]
There is a test case with carriage return. I've actually included small .csv files.

**Comment by chr1sgrant** [comment]
Ahh I meant just to add a test example which has the example with carriage returns scoped within the existing test list for TestParsePlate. The added test looks more appropriate for the github.com/antha-lang/toolbox package as a direct unit test of the NewTolerantReader() func

**Comment by cosmonautspiff** [comment]
@chr1sgrant I've updated the parse_plate test as we discussed. It reads in an external .csv file in order to preserve the carriage returns.

**Pull request merged**